constantine/math/constants/banderwagon_sqrt.nim

```diff
@@ -18,6 +19,1347 @@ const
   Banderwagon_TonelliShanks_twoAdicity* = 32
   Banderwagon_TonelliShanks_root_of_unity* = Fp[Banderwagon].fromHex"0x212d79e5b416b6f0fd56dc8d168d6c0c4024ff270b3e0941b788f500b912f1f"
 
+  Banderwagon_SqrtDlog_TotalBits* = Banderwagon_TonelliShanks_twoAdicity
```

Comment:
This part should be below the `Specialized Tonelli-Shanks` section

and with a title

```Nim
# ############################################################
#
#       Optimized square-root via Discrete Log lookup tables
#
# ############################################################
```